app/views/user_mentee_applications/mentee_application_states/create.turbo_stream.erb

```diff
@@ -17,3 +17,13 @@
     %>
   <% end %>
 <% end %>
+
+<%= turbo_stream.update "#{dom_id(@user_mentee_application)}--manage" do %>
```

Comment:
it's a little bit of duplication, but more explicit and simpler than having to leverage i18n in this case.